### DISCUSSION TITLE
Question: What is the best way to cooperate with non-technical colleagues?

### DISCUSSION THREAD

**Discussion by sqandrew** [pytest-dev/pytest-bdd#602] [General]
What tool or approach do you use for cooperation with guys who don't use IDE?

For example to get someone from a business involved in feature creation from already existing steps (autocomplete).

Cucumber has cucumber-studio, and what can be used to simplify that activity for pytest-bdd lovers?

**Comment by youtux** [comment]
I'm afraid I don't really have an answer for this. I use pytest-bdd as a developer, and I translate requirements to BDD scenarios myself.

**Comment by blaisep** [comment]
@sqandrew , I'm super interested in this topic.
Ironically, we don't use cucumber-studio in 
https://youtu.be/Yetx2eOuP9w?si=823f89lovGiFUZ2x

**Comment by blaisep** [comment]
My suggestion would be to use Sphinx and include the feature files in the docs.
You can then associate the feature files with the source code and with the step functions.
